Resources/Prototypes/_Impstation/Entities/Mobs/Customization/Markings/human_hair.yml

```
@@ -21,3 +21,11 @@
   sprites:
     - sprite: _Impstation/Mobs/Customization/human_hair.rsi
       state: frenchbraid
+
+- type: marking
+  id: HumanHairMessyPonytail
+  bodyPart: Hair
+  markingCategory: Hair
+  sprites:
+    - sprite: _Impstation/Mobs/Customization/human_hair.rsi
+      state: messyponytail
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)